llvm/test/lab2/kulikov_artem/3for.ll

```diff
@@ -0,0 +1,150 @@
+; RUN: opt -passes=kulikov-wrap-plugin -S %s | FileCheck %s
```

Comment:
Could you please load this plugin dynamically via `-load-pass-plugin` option of opt?

Reply:
This plugin is built in static library. We can't load it

Reply:
Making it dynamic is part of the comment.